classes/services/IssueService.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * @file classes/services/IssueService.php
+*
+* Copyright (c) 2014-2017 Simon Fraser University
+* Copyright (c) 2000-2017 John Willinsky
+* Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+*
+* @class IssueService
+* @ingroup services
+*
+* @brief Helper class that encapsulates issue business logic
+*/
+
+namespace App\Services;
+
+class IssueService {
+
+	/**
+	 * Determine if a user can access galleys for a specific issue 
+	 * 
+	 * @param \Journal $journal
+	 * @param \Issue $issue
+	 * 
+	 * @return boolean
+	 */
+	public function userHasAccessToGalleys(\Journal $journal, \Issue $issue) {
```

Comment:
I'm rewriting the subscription stuff now and may break it up into classes like this -- but for now, would it be possible to keep all that code in `IssueAction` where it's currently centralized?